src/map/lua/lua_baseentity.cpp

```diff
@@ -2067,7 +2067,7 @@ inline int32 CLuaBaseEntity::getStat(lua_State *L)
         case MOD_INT:  lua_pushinteger(L, PEntity->INT()); break;
         case MOD_MND:  lua_pushinteger(L, PEntity->MND()); break;
         case MOD_CHR:  lua_pushinteger(L, PEntity->CHR()); break;
-        case MOD_ATT:  lua_pushinteger(L, PEntity->ATT()); break;
+        case MOD_ATT:  lua_pushinteger(L, PEntity->ATT(SLOT_MAIN, 0)); break;
```

Comment:
we are definitely going to have to expose the slot to Lua and have the scripts check it